### PR TITLE
Added LTS in ToC

### PR DESCRIPTION
**Pull request by lmammino**
Ref https://github.com/fastify/website/issues/55

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
You'll need to remove https://github.com/fastify/fastify#long-term-support too.

File: README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ matters to you.
 * <a href="https://github.com/fastify/fastify/blob/master/docs/Benchmarking.md"><code><b>Benchmarking</b></code></a>
 * <a href="https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md"><code><b>Plugins Guide</b></code></a>
 * <a href="https://github.com/fastify/fastify/blob/master/docs/HTTP2.md"><code><b>HTTP2</b></code></a>
+* <a href="https://github.com/fastify/fastify/blob/master/docs/LTS.md"><code><b>LTS</b></code></a>
```
Comment (mcollina) [inline README.md]: call it "long term support"

**Comment by lmammino** [comment]
Thanks for the ultra-quick review @mcollina. 

Do you generally wait for more than one reviewer before merging?

**Comment by mcollina** [comment]
This can land as soon as CI is happy.

**Pull request merged**